src/Writers/HTMLWriter.jl

```diff
@@ -1072,19 +1071,29 @@ function render_navbar(ctx, navnode, edit_page_link::Bool)
         host_type = Utilities.repo_host_from_url(ctx.doc.user.repo)
         if host_type == Utilities.RepoGitlab
             host = "GitLab"
-            logo = "\uf296"
+            logo = svg[:viewBox => "0 0 512 512", :xlmns=>"http://www.w3.org/2000/svg"](
```

Comment:
Where are these taken from? Would be nice to add a comment with a URL or something.